web-src/src/router/index.js

```diff
@@ -128,6 +131,24 @@ export const router = new VueRouter({
       component: PageGenreTracks,
       meta: { show_progress: true, has_index: true }
     },
+    {
+      path: '/music/composers',
+      name: 'Composers',
+      component: PageComposers,
+      meta: { show_progress: true, has_tabs: true, has_index: true }
+    },
+    {
+      path: '/music/composers/:composer/albums',
+      name: 'ComposerAlbums',
+      component: PageComposer,
+      meta: { show_progress: true, has_tabs: true, has_index: true }
+    },
+    {
+      path: '/music/composers/:composer/tracks',
```

Comment:
composers arent analogous to an artist - a composer may be featured on many albums so its easier fot this to presnt tracks by default since those ARE the ones that belong to the compser